fastlane_core/spec/configuration_spec.rb

```diff
@@ -804,14 +804,44 @@
             ENV.delete("abc")
           end
 
-          it "prioritizes config file values after ENV" do
+          it "prioritizes env_names after CLI" do
+            ENV["abc"] = "val env"
```

Comment:
you might want to use `with_env_values` to avoid having to set / reset the environment. 

maybe for a future cleanup. I see  it uses this pattern elsewhere already.